ir_ui_view_multi_company/models/ir_ui_view.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.osv.expression import AND
+
+
+class IrUiView(models.Model):
+    _inherit = "ir.ui.view"
+    _inheriting_views_domain_company_operator = (
+        "="  # Override this if needed, e.g., use 'parent_of' or 'child_of'
+    )
```

Comment:
Maybe add a note in the README so that we know without reading the code that these options are easily activable.

Reply:
done